src/index.html

```diff
@@ -1,13 +1,63 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div data-qa="card" class="card">
+      <div class="card__product_image">
+        <img
+          class="card__product_image_img"
+          src="./images/imac.jpeg"
+          alt="Моноблок APPLE A1419 iMac 27"
+        >
+      </div>
+      <div class="card__product_name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="card__product_code">
+        Код товара: 195434
+      </div>
```

Comment:
```suggestion
      <div class="card__product_code">
        Код товара: 195434
      </div>
```
Використовуй для текстової інформації теги p чи span